libusb/os/windows_winusb.c

```diff
@@ -2607,6 +2605,27 @@ static int winusbx_configure_endpoints(int sub_api, struct libusb_device_handle
 	return LIBUSB_SUCCESS;
 }
 
+static HANDLE winusbx_ensure_dev_handle(struct libusb_device_handle *dev_handle, struct winusb_device_priv *priv, struct winusb_device_handle_priv *handle_priv, uint8_t iface)
+{
```

Comment:
And should it rather resolve the handle_priv and priv from dev_handle instead of passing them as a function arguments? The function interface should be non-redudant. The accessor functions are practically no-cost.

Reply:
👍